cpp/src/plasma/eviction_policy.cc

```diff
@@ -67,32 +67,24 @@ int64_t EvictionPolicy::choose_objects_to_evict(int64_t num_bytes_required,
 void EvictionPolicy::object_created(const ObjectID& object_id) {
   auto entry = store_info_->objects[object_id].get();
   cache_.add(object_id, entry->info.data_size + entry->info.metadata_size);
+  int64_t size = entry->info.data_size + entry->info.metadata_size;
+  memory_used_ += size;
```

Comment:
I moved this from `require_space` to `object_created`. We need to do this bookkeeping for every object, but `require_space` is only called when there isn't enough space.